### ISSUE TITLE
Adjust package versions to latest floating patch

### ISSUE THREAD

**Issue by benbrown** [microsoft/botframework-components#1118]
The generators should use a floating patch version, equivalent to the `~major.minor.minpatch` semver syntax so that they will automatically use the latest patch release version of the SDK and components.

This will enable us to release bug fixes of the SDK and components without having to ship the entire generator library each time.

- [ ] Dotnet webapp csproj
- [ ] Dotnet functions csproj
- [ ] Dotnet yeoman generator code
- [ ] Javascript yeoman generator code

In addition, do we also need to apply this to the specific generators and components? There are specific versions currently used wherever there are dependencies - People, Calendar, core, etc.

**Comment by tracyboehrer** [comment]
Yes, but... According to Josh G, this would not work with the use of "-preview" in package versions.  eg. ~4.13.3-preview would not match 4.13.4-preview.

If we could solve that, it would make all this work better though.

**Comment by joshgummersall** [comment]
@tracyboehrer I think we should revisit this in the JS SDK.

**Comment by sgellock** [comment]
Shiproom - moving out of R13.  we need GA versions of the JS SDK before we can address this issue.
